ocs_ci/helpers/disruption_helpers.py

```diff
@@ -22,9 +23,38 @@ class Disruptions:
     resource_count = 0
     selector = None
     daemon_pid = None
+    cluster_kubeconfig = ""
+
+    def kubeconfig_parameter(self):
+        """
+        Returns the '--kubeconfig <value>' parameter for the oc command
+
+        Returns:
+            str: The '--kubeconfig <value>' parameter for oc command if the attribute 'cluster_kubeconfig' is not empty.
+                Empty string if the the attribute 'cluster_kubeconfig' is empty.
+        """
+        kubeconfig_parameter = (
+            f"--kubeconfig {self.cluster_kubeconfig} "
+            if self.cluster_kubeconfig
+            else ""
+        )
+        return kubeconfig_parameter
 
     def set_resource(self, resource, leader_type="provisioner"):
         self.resource = resource
+        if (config.ENV_DATA["platform"] in constants.MANAGED_SERVICE_PLATFORMS) and (
+            resource in CEPH_PODS
+        ):
+            # If the platform is Managed Services, then the ceph pods will be present in the provider cluster.
+            # Consumer cluster will be the primary cluster context in a multicluster run. Setting 'cluster_kubeconfig'
+            # attribute to use as the value of the parameter '--kubeconfig' in the 'oc' commands to get ceph pods.
+            provider_kubeconfig = os.path.join(
+                config.clusters[config.get_provider_index()].ENV_DATA["cluster_path"],
+                config.clusters[config.get_provider_index()].RUN.get(
+                    "kubeconfig_location"
+                ),
+            )
+            self.cluster_kubeconfig = provider_kubeconfig
```

Comment:
What about name this parameter `provider_kubeconfig` to make it clear to which cluster it belongs to?

Reply:
cluster_kubeconfig can have provider or consumer cluster kubeconfig. This PR address tests cases which use provider cluster kubeconfig.